geomesa-gs-plugin/geomesa-kafka-gs-plugin/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependencies>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-kafka-datastore</artifactId>
+            <artifactId>geomesa-kafka-10-datastore</artifactId>
```

Comment:
Maybe we should explicitly have three gs plugins (``geomesa-kafka-{08,09,10}-gs-plugin``) to match the changes to the KDS.